packages/components/psammead-radio-schedule/src/ProgramCard/index.jsx

```diff
@@ -275,11 +269,10 @@ const ProgramCard = ({
       </IconWrapper>
       <DurationWrapper dir={dir} dateTime={duration}>
         <VisuallyHiddenText>
-          {` ${durationLabel} ${formatDuration({
-            duration,
-            format: getDurationFormat(duration, ','),
-            locale,
-          })} `}
+          {`${detokenise(
+            ` ${durationLabel}`,
```

Comment:
Do you need both of these nested interpolated strings? Can you simplify this so there is just one?

Reply:
This is to handle the fact that we need a space before `Duration: %duration%`. This is needed when CSS and Javascript are disabled:
![image](https://user-images.githubusercontent.com/53564281/80619203-4778ee80-8a3c-11ea-9ffd-b72466d6c5dc.png)
If we didn't do this, I'm not sure how we would add the space in. If we don't then I think we might have a UX problem:
![image](https://user-images.githubusercontent.com/53564281/80619292-6d05f800-8a3c-11ea-8ace-1a219139a179.png)

Reply:
Could you change the `  ${durationLabel}` to just be durationLabel, and add the extra space before the detokenise function? Then you only need one interpolation

Reply:
Ah, nice catch. Completely missed that it was all already in a template literal. Fixed now, thanks.